GVFS/GVFS.FunctionalTests/Tests/GitCommands/StatusTests.cs

```diff
@@ -136,7 +137,13 @@ public void ModifyingHeadRefInvalidatesCache()
         [TestCase]
         public void AppendFile()
         {
-            this.AppendAllText("More Data", @"Readme.md");
+            BashRunner bash = new BashRunner();

```

Comment:
Minor: Could add some comments here describing why we're using Bash specifically?  It might also be worth updating the test name.